app/assets/stylesheets/home/index.css

```diff
@@ -3,3 +3,15 @@ path {
   stroke-width: 1.00px;
   fill: #f8f4e8;
 }
+
```

Comment:
We'll probably get the styling for buttons next week...So this will probably change anyway.